README.it_IT.md

```diff
@@ -0,0 +1,83 @@
+<p align="center">
+<img src="https://media.githubusercontent.com/media/rocky-linux/branding/main/logo-text-light%402x.png" alt="Rocky Linux Logo">
+</p>
+
+<p align="center">
+<i>"Ripensando ai primi giorni di CentOS... Il mio co-fondatore fu Rocky McGaugh. Lui non è più con noi, quindi in suo ricordo e rispetto, dato che non ha potuto vedere quanto successo CentOS ha riscosso, vi presento...Rocky Linux"</i><br>
+— Gregory Kurtzer, Fondatore
+</p>
+
+## About
+
+Rocky Linux è un sistema operativo enterprise basato sulla community e progettato per essere 100% compatibile bug-per-bug con Enterprise Linux, ora che CentOS ha cambiato direzione.
+
+## Domande frequenti (Frequently Asked Questions)
+
+> **Q:** In che senso, "CentOS ha cambiato direzione?"
+
+Il progetto CentOS [ha recentemente annunciato](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio di strategia per CentOS. Laddove precedentemente CentOS esisteva come *downstream* build del suo vendor (riceveva patch e aggiornamenti dopo l'upstream vendor), adesso diventerà una *upstream* build (testing patches e aggiornamenti ricevuti *prima* dell'upstream vendor).
+
+Inoltre, il supporto per CentOS Linux 8 è stato accorciato, dal 31 Maggio 2029 al 31 Dicembre 2021.
+
+
+> **Q:** Quindi come si posiziona Rocky Linux?
+
+Rocky Linux punta ad essere una downstream build come lo è stato CentOS in precedenza, effettuando rilasci dopo l'upstream vendor, non prima.
+
+> **Q:** Quando sarà rilasciato?
+
+Non c'è ancora un ETA per il rilascio.
+
+> **Q:** Qual è la vision per Rocky Linux?
+
+Un'alternativa **solida**, **stabile**, e **trasparente** per gli ambienti di produzione, sviluppata *dalla* community *per* la community.
+
+> **Q:** Chi guida lo sviluppo di Rocky Linux?
+
+Noi tutti, Rocky Linux è un progetto community-driven e lo sarà per sempre. Rocky Linux non sarà venduto o guidato da interessi aziendali.
+
+> **Q:** Come posso partecipare?
+
+Per piacere leggi la sezione [contribuire](#contribuire) in fondo.
+
+## Contatti
+
+| Team                          | Contatto                                  |
+|-------------------------------|-------------------------------------------|
+| Stampa                        | outreach@rockylinux.org                   |
```

Comment:
Maybe instead of `Stampa` an `Ufficio Stampa`

Reply:
Absolutely, i'm merging that.